containers/nginx-storage/default.conf

```diff
@@ -0,0 +1,12 @@
+error_log logs/error.log notice;
+
+server {
+    # HTTP
+    listen 81       default_server;
```

Comment:
This config does not include `file-transfer-endpoints.conf`. How does it provide the `upload/` and `download/` endpoints?

Reply:
It's include `file-transfer-endpoints.conf`. I don't understand what the problem is ?

Reply:
I must have been looking at a specific commit. It looks OK now.